menpowidgets/base.py

```diff
@@ -673,6 +673,9 @@ def visualize_images(images, figure_size=(10, 8), style='coloured',
     browser_style : ``{'buttons', 'slider'}``, optional
         It defines whether the selector of the objects will have the form of
         plus/minus buttons or a slider.
+    messages: `list` with the same length as the `images`, optional
```

Comment:
This should be
```
messages : `list` or ``None``, optional
```
And then explain that it must have the same length as `images` in the description below.